src/components/tasks/taskStrip.tsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import strip_style from "./taskStrip.module.sass";
+import ReactMarkdown from "react-markdown";
+import SyntaxHighlighter from "react-syntax-highlighter";
+
+function Strip(props: any) {
+  const handleExtend = (e: React.MouseEvent) => {
```

Comment:
This allows us to expand the taskStrip component to allow us to display the tasks description.